src/main/java/net/sourceforge/pmd/util/fxdesigner/DesignerStarter.java

```diff
@@ -92,7 +101,7 @@ private static String getHelpText(JCommander jCommander) {
     }
 
     @SuppressWarnings("PMD.AvoidCatchingThrowable")
-    private static void launchGui(String[] args) {
+    public static int launchGui(String[] args) {
```

Comment:
If this is now public, then the constants ERROR_EXIT and OK should also be. In fact it would be better to publish a small enum I think